libraries/AP_Scripting/applets/MissionSelector.lua

```diff
@@ -0,0 +1,88 @@
+-- Loads one of three mission files to autopilot on each arm, depending on position of the Mission Reset AUX FUNC switch
+-- Must have Mission Reset switch assigned, it will function normally when armed or disarmed
+-- but also on the disarm to arm transition, it will load (if file exists) a file in the root named
+-- missionH.txt, missionM.txt, or missionH.txt corresponding to the the Mission Reset switch position of High/Mid/Low
+
+local mission_loaded = false
+local rc_switch = rc:find_channel_for_option(24)  --AUX FUNC sw for mission restart
+
+if not rc_switch then  -- requires the switch to be assigned in order to run script
+  return
```

Comment:
You could add something like
`error('Mission restart switch no setup')`

Reply:
but the instructions make it clear.

Reply:
specifically did not want to do that, so it could be on the card if there was no switch available at the moment....ie just put it on all the planes, and if you want to use it, setup the switch, enable scripting and add the missions...meant to fail silently

Reply:
not married to it, but on my setups I would prefer it not spamming me while other scripts are running if I am not using it